fe/projects/Seller.Portal/src/shared/helpers/forms/useForm.js

```diff
@@ -135,7 +135,12 @@ function useForm(
       setIsDirty(true);
 
       if (event.isFormData) {
-        setFieldValue({ name: "formData", value: { ...event.formData, [event.name]: event.target.value } });
+        if(event.target.type === "number") {
```

Comment:
**suggestion:** it can be nicely put into the setFieldValue call so that there is only one cass of setFieldValue. Try to use GPT for inspiration